dotCMS/src/main/java/com/dotmarketing/portlets/templates/design/bean/ContainerHolder.java

```diff
@@ -36,16 +38,19 @@ public String draw () throws Exception {
         final StringBuilder sb = new StringBuilder();
 
         if ( this.containers != null ) {
-            for ( final String container: this.containers ) {
+            for ( final ContainerUUID container: this.containers ) {
 
                 if ( this.preview ) {
                     sb.append( PreviewTemplateUtil.getMockBodyContent() );
                 } else {
-                    sb.append( "#parseContainer('" ).append( container ).append( "')" );
+                    String uuid = container.getUUID() != null ? container.getUUID() : ParseContainer.DEFAULT_UUID_VALUE;
```

Comment:
This could be a method on StringUtils.getOrDefault (container.getUUID(), ()->ParseContainer.DEFAULT_UUID_VALUE);

String getOrDefault (final String string, final Supplier<String> defaultSupplier) {
    return string != null ? string : defaultSupplier.get();
}